CHANGELOG.md

```diff
@@ -272,7 +274,7 @@ Some major changes worth highlighting:
         - No changes other than general cross-crate changes
     - `icu_locale`
         - `LocaleExpander`, `LocaleDirectionality`, and `LocaleCanonicalizer` distinguish between `new_common()` and `new_extended()` constructors (unicode-org#5958)
-    - `icu_locale_core` 
+    - `icu_locale_core`
```

Comment:
please revert formatting-only changes. any formatting that's not enforced in CI is just going to break again